app/views/application_records/show.haml

```diff
@@ -6,6 +6,10 @@ by:
 = @record.user.name_and_email
 = button_to 'Print this page',
   application_record_path(@record, format: :pdf), method: :get
+- if !@record.saved_for_later && @record.note_for_later.present?
```

Comment:
> You can't query an object based on an attribute they previously had.

Not entirely true, we could enable `paper_trail`.

But of course, isn't the presence of a `note_for_later` sufficient to assume that it was once saved for later?